docs/contracts/overview.md

```diff
@@ -73,14 +76,28 @@ transaction independently of any Buyer action.
    the quantity of available things that all bear similar properties, we say 
    such an offer is a Voucher Set.  
 
+```solidity
+BosonRouter.requestCreateOrderETHETH()  
+```
+   
+   Alternatively, the Seller can make an offer with an associated gate contract and
+   non-transferrable NFT Id. In this case, the Seller is making the item(s) on offer
+   only available to Buyers who execut a conditional commit. The "condition" of
```

Comment:
"execut" -> "execute"

Reply:
Changed. Will be included in next commit.